occlum/run_one_tests_on_occlum.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Give some info if the user doesn't input anything.

Reply:
Changes have been submitted